api/app/Admin.js

```diff
@@ -61,4 +61,32 @@ module.exports = function() {
             console.log(e);
         }
     });
+
+    //MongoDb Native Access
+    global.app.post('/admin/dbaccess/enable/:appId',function(req, res) {
```

Comment:
Very very important : 

Comments on the function should be in this format : 

/**
*Description : 
*Params: 
*- Param 1 : Description of param1
*- Param N: Desc of param n
*Returns:
-Success : Desc of what is being returned on success
-Error : Desc of what is being returned on error
*/